heksher/setting_type.py

```diff
@@ -163,18 +181,22 @@ def __init__(self, inner: SettingType[T]):
     def heksher_string(self) -> str:
         return f'Sequence<{self.inner.heksher_string()}>'
 
-    def convert(self, x) -> Conversion[Sequence[T]]:
+    def convert_from_heksher(self, x) -> Conversion[Sequence[T]]:
         values = []
         coercions = []
         for i, v in enumerate(x):
             try:
-                conversion = self.inner.convert(v)
+                conversion = self.inner.convert_from_heksher(v)
             except TypeError as e:
                 coercions.append(f'failed to convert element {i}: {e!r}')
             else:
                 values.append(conversion.value)
                 coercions.extend(f'element {i}: {c}' for c in conversion.coercions)
-        return Conversion(tuple(values), coercions)
+        ret = Conversion(tuple(values), coercions)
+        return ret
```

Comment:
?

Reply:
Oy vey, I did it for printing and forgot to change back